### PR TITLE
Missing unit tests

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

[DIV-3424 - One Per Page Test Suite - Add missing tests](https://tools.hmcts.net/jira/browse/DIV-3424)

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ const govukTemplate = require('@hmcts/look-and-feel/src/sources/govukTemplate');
 const lookAndFeel = require('@hmcts/look-and-feel/src/sources/lookAndFeel');
 const { expect } = require('../utils/chai');
 const httpStatus = require('http-status-codes');
-const { get } = require('lodash');
+const { get, unset } = require('lodash');
```
Comment (hosainnet) [inline src/content.js]: unset is unused, are we're missing a lint rule?

File: test/src/custom.test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const { custom, stepAsInstance } = require(modulePath);
+const chai = require('chai');
+const SampleStep = require('../steps/exit/Sample.step');
+const { testStepDSL } = require('utils/supertest');
+
+const expect = chai.expect;
+
+describe(modulePath, () => {
+  it('#custom returns test step DSL with instansiated with step', () => {
```
Comment (alexwakeman) [inline test/src/custom.test.js]: Typo, repeated `with`

File: test/src/journey.test.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+const modulePath = 'src/journey';
```
Comment (alexwakeman) [inline test/src/journey.test.js:1]: Should this journey test include an `Exit` step?

**Review by alexwakeman — APPROVED** [review]

File: src/navigation.js [hunk 4]
```diff
@@ -2,18 +2,26 @@ const { testStep } = require('../utils/supertest');
...
     test.withSession(session);
-  } else {
+  } else if (stepType !== 'EntryPoint'){
     test.withSetup(req => {
       return req.session.generate();
     });
   }
 
-  return test.post()
+  if (stepType === 'Redirect' || stepType === 'EntryPoint') {
```
Comment (hosainnet) [inline src/navigation.js:18]: Personally, I would prefer having specific functions with an appropriate name for kind of step but maybe this is more backward compatible

File: src/question.js [hunk 5]
```diff
@@ -13,6 +13,7 @@ const templates = [
   'mocks',
   'mocks/steps',
   'steps',
+  'test/steps',
   'node_modules/reform-pattern-library/app/views/macros'
```
Comment (hosainnet) [inline src/question.js]: Not part of this, but this should go 😅

**Review by hosainnet — APPROVED** [review]
Minor comments, but happy assuming it's has been tested with DN/RFE and requires no changes there (given it's a x.x.1 patch)

File: src/navigation.js (hunk 4 above)
Comment (dtaylor7) [reply src/navigation.js:18]: i thought it better just to have one test for navigation that can handle each type of step, given this is the only difference...

File: src/question.js (hunk 5 above)
Comment (dtaylor7) [reply src/question.js]: 👍

File: src/content.js (hunk 1 above)
Comment (dtaylor7) [reply src/content.js]: 👍

File: test/src/journey.test.js (hunk 3 above)
Comment (dtaylor7) [reply test/src/journey.test.js:1]: 👍

File: test/src/custom.test.js (hunk 2 above)
Comment (dtaylor7) [reply test/src/custom.test.js]: 👍

**Pull request merged**